lib/yardstick/parser.rb

```diff
@@ -10,7 +10,7 @@ class Parser
     #
     # @api private
     def self.parse_paths(paths)
-      YARD.parse(paths, [], YARD::Logger::ERROR)
+      YARD.parse(paths, [], Logger::ERROR)
```

Comment:
Yeah `Yard::Logger` inherits from `Logger` so the `ERROR` const for both are the same